src/desdemona/ui/table.cljs

```diff
@@ -2,7 +2,7 @@
   (:require [wilson.dom :as wd]
             [reagent.session :as session]
             [dommy.core :as d :refer-macros [sel sel1]]
-            [desdemona.ui.dom :refer [get-el-height get-el-width]]
+            [desdemona.ui.dom :as dd :refer [get-el-height get-el-width]]
```

Comment:
They are used in multiple places. I guess the `:refer` is not necessary, the names was just long enough for my taste